bindings/rust/bench/src/s2n_tls.rs

```diff
@@ -0,0 +1,184 @@
+use crate::{
+    harness::{Buffer, Mode, TlsImpl},
+    read_to_bytes, CA_CERT_PATH, SERVER_CERT_CHAIN_PATH, SERVER_KEY_PATH,
+};
+use log::info;
+use s2n_tls::{
+    callbacks::VerifyHostNameCallback,
+    config::{Builder, Config},
+    connection::Connection,
+    enums::Blinding,
+    security::DEFAULT_TLS13,
+};
+use std::{
+    ffi::c_void,
+    io::{Read, Write},
+    os::raw::c_int,
+    pin::Pin,
+    task::Poll::Ready,
+};
+
+pub struct S2nTls {
```

Comment:
Nit: the capital `S` on s2n-tls looks weird to me? I wonder if there is a better name. Perhaps `Signal`? `SignalToNoise`?

Reply:
I don't know what else to change it to, but I'm open to suggestions. I think `Signal` and `SignalToNoise` would be confusing because most people probably don't know what s2n stands for.

Reply:
Changed to SignalToNoise, should look cleaner